### PR TITLE
Table improvements (colors, internal tx)

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Comment by LeeAlephium** [comment]
@nop33 feel free to change anything I've done; at its core all that matters is the Move logic is completed now

File: locales/en-US/App.json [hunk 1]
```diff
@@ -232,5 +232,6 @@
   "into": "into",
   "out from": "out from",
   "Transaction details": "Transaction details",
-  "Copy address": "Copy address"
+  "Copy address": "Copy address",
+  "Moved": "Moved"
```
Comment (nop33) [inline locales/en-US/App.json:236]: We need this in French

Comment (mvaivre) [reply locales/en-US/App.json:236]: I'll add those to #371 is it's fine by you, just to avoid going through merge conflicts.

**Review by LeeAlephium — APPROVED** [review]
Just 2 nits otherwise we're good to go :+1:

File: src/components/TransactionalInfo.tsx [hunk 2]
```diff
@@ -44,81 +51,129 @@ interface TransactionalInfoProps {
...
         <TokenTimeInner>
+          {infoType === 'move'
+            ? t`Moved`
+            : infoType === 'pending'
+            ? t`Pending`
+            : infoType === 'out'
+            ? t`Sent`
+            : infoType === 'in'
+            ? t`Received`
+            : null}
```
[the remark this answers is not in the document]
Comment (nop33) [reply src/components/TransactionalInfo.tsx]: Fixed!

**Review by LeeAlephium — APPROVED** [review]

**Comment by nop33** [comment]
Rebased

**Comment by LeeAlephium** [comment]
Will merge when checks pass :+1:

**Pull request merged**